packages/contracts-bedrock/test/kontrol/proofs/utils/DeploymentSummaryCode.sol

```diff
@@ -28,8 +28,12 @@ contract DeploymentSummaryCode {
+    bytes internal constant L1CrossDomainMessengerProxyCode =
```

Comment:
This is a false positive, since that constant is used [here](https://github.com/runtimeverification/_audits_Ethereum-optimism_pausability/blob/8926b7676de7f5fa4b8f779d4133d94e075c101e/packages/contracts-bedrock/test/kontrol/proofs/utils/DeploymentSummary.sol#L159)